### ISSUE TITLE
Swag can not create root level `produce` and `consume` spec entries

### ISSUE THREAD

**Issue by spkane** [swaggo/swag#887]
**Describe the bug**
It should be possible to create the `accepts:` and `consumes` entries at the root level of the swagger spec as discussed in:
* https://swagger.io/docs/specification/2-0/mime-types/
* https://swagger.io/docs/specification/2-0/basic-structure/

However using,

```go
// @Accept  json
// @Produce  json
```

in the General section, alongside things like `@Schemes`, seems to be ignored.

**To Reproduce**
Have a General section that looks like this:

```go
// @title Example API
// @version 1.0.1
// @description This is a sample API for some purpose.

// @contact.name Example
// @contact.url http://example.com/
// @contact.email info@example.com

// @BasePath /

// @Schemes http https

// @securityDefinitions.apikey ApiKeyAuth
// @in header
// @name Authorization

// @Accept  json
// @Produce  json
```

**Expected behavior**
The resulting top level of the `swagger.yaml` has these sections:

```yaml
consumes:
  - application/json
produces:
  - application/json
```

**Your swag version**
1.7.0

**Your go version**
1.15.7

**Desktop (please complete the following information):**
 - OS: macOS 11.2.1
 - Browser: Chrome
 - Version: 88.0.4324.150 (Official Build) (x86_64)

**Comment by ubogdan** [comment]
@spkane This is not implemented and it may be a nice to have feature because it will reduce the annotation boilerplate.

**Comment by ubogdan** [comment]
Feature merged, it will be available on v1.7.4.
